### ISSUE TITLE
Repair this thing

### ISSUE THREAD

**Issue by kittech0** [MerchantPug/apugli#24]
```
---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 10/2/22, 9:41 PM
Description: Unexpected error

java.lang.ClassCastException: class net.minecraft.class_745 cannot be cast to class com.github.merchantpug.apugli.mixin.client.ClientPlayerEntityAccessor (net.minecraft.class_745 and com.github.merchantpug.apugli.mixin.client.ClientPlayerEntityAccessor are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @21a947fe)
	at com.github.merchantpug.apugli.condition.entity.ParticleInRadiusCondition.condition(ParticleInRadiusCondition.java:34)
	at io.github.apace100.apoli.power.factory.condition.ConditionFactory$Instance.isFulfilled(ConditionFactory.java:43)
	at io.github.apace100.apoli.power.factory.condition.ConditionFactory$Instance.test(ConditionFactory.java:34)
	at io.github.apace100.apoli.power.Power.lambda$isActive$0(Power.java:79)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
	at java.base/java.util.LinkedList$LLSpliterator.tryAdvance(LinkedList.java:1256)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.allMatch(ReferencePipeline.java:637)
	at io.github.apace100.apoli.power.Power.isActive(Power.java:79)
	at io.github.apace100.apoli.component.PowerHolderComponentImpl.getPowers(PowerHolderComponentImpl.java:73)
	at io.github.apace100.apoli.component.PowerHolderComponentImpl.getPowers(PowerHolderComponentImpl.java:65)
	at io.github.apace100.apoli.component.PowerHolderComponent.withPower(PowerHolderComponent.java:101)
	at net.minecraft.class_757.handler$zeh000$loadShaderFromPower(class_757.java:1578)
	at net.minecraft.class_757.method_3192(class_757.java)
	at net.minecraft.class_310.method_1523(class_310.java:1143)
	at net.minecraft.class_310.method_1514(class_310.java:734)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at com.github.merchantpug.apugli.condition.entity.ParticleInRadiusCondition.condition(ParticleInRadiusCondition.java:34)
	at io.github.apace100.apoli.power.factory.condition.ConditionFactory$Instance.isFulfilled(ConditionFactory.java:43)
	at io.github.apace100.apoli.power.factory.condition.ConditionFactory$Instance.test(ConditionFactory.java:34)
	at io.github.apace100.apoli.power.Power.lambda$isActive$0(Power.java:79)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
	at java.base/java.util.LinkedList$LLSpliterator.tryAdvance(LinkedList.java:1256)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.allMatch(ReferencePipeline.java:637)
	at io.github.apace100.apoli.power.Power.isActive(Power.java:79)
	at io.github.apace100.apoli.component.PowerHolderComponentImpl.getPowers(PowerHolderComponentImpl.java:73)
	at io.github.apace100.apoli.component.PowerHolderComponentImpl.getPowers(PowerHolderComponentImpl.java:65)
	at io.github.apace100.apoli.component.PowerHolderComponent.withPower(PowerHolderComponent.java:101)
	at net.minecraft.class_757.handler$zeh000$loadShaderFromPower(class_757.java:1578)

-- Affected level --
Details:
	All players: 2 total; [class_746['0Cute'/201018, l='ClientLevel', x=405.71, y=62.07, z=-26.23], class_745['Xenpheric'/201127, l='ClientLevel', x=402.34, y=62.07, z=-26.45]]
	Chunk stats: 256, 94
	Level dimension: minecraft:overworld
	Level spawn location: World: (-198,68,77), Section: (at 10,4,13 in -13,4,4; chunk contains blocks -208,-64,64 to -193,319,79), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,-64,0 to -1,319,511)
	Level time: 275004 game time, 275004 day time
	Server brand: fabric (Velocity)
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:453)
	at net.minecraft.class_310.method_1587(class_310.java:2288)
	at net.minecraft.class_310.method_1514(class_310.java:758)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, Supplementaries Generated Pack

-- System Details --
Details:
	Minecraft Version: 1.19
	Minecraft Version ID: 1.19
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1085557248 bytes (1035 MiB) / 3934257152 bytes (3752 MiB) up to 5704253440 bytes (5440 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz
	Identifier: Intel64 Family 6 Model 94 Stepping 3
	Microarchitecture: unknown
	Frequency (GHz): 3.19
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: NVIDIA GeForce GTX 960
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1401
	Graphics card #0 versionInfo: DriverVersion=31.0.15.1748
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.40
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 17357.65
	Virtual memory used (MB): 12246.42
	Swap memory total (MB): 9216.00
	Swap memory used (MB): 1157.59
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx5440m -Xms256m
	Fabric Mods: 
		adventurez: AdventureZ 1.4.14
		all_the_fan_made_discs: All The Fan Made Discs 0.3.4
		apocalypse: Apocalypse Origins 0.2.27
		appleskin: AppleSkin 2.4.1+mc1.19
		architectury: Architectury 5.12.45
		attributefix: AttributeFix 15.1.3
		axesareweapons: Axes Are Weapons 1.6.0
		bookshelf: Bookshelf 14.2.10
		capybara: Capybara 1.0.6
		chat_heads: Chat Heads 0.6.0
		cloth-config: Cloth Config v8 8.2.88
			cloth-basic-math: cloth-basic-math 0.6.1
		controlling: Controlling For Fabric 10.0+3
		customizableelytra: Customizable Elytra 1.6.2-1.19
		disable_custom_worlds_advice: Disable Custom Worlds Advice 3.0
		dramaticdoors: Dramatic Doors (Fabric) 1.19.2-1.3.1
		dungeons_arise: When Dungeons Arise 2.1.51
		dynamicsoundfilters: Dynamic Sound Filters 1.3.0+1.19
		effective: Effective 1.3
			midnightlib: MidnightLib 0.5.2
		eldritch_mobs: EldritchMobs 1.12.6
			cardinal-components-base: Cardinal Components API (base) 5.0.0-beta.1
			cardinal-components-entity: Cardinal Components API (entities) 5.0.0-beta.1
		enchant_the_rainbow: Enchant the Rainbow 1.1.0
		enchdesc: EnchantmentDescriptions 11.1.9
		extraorigins: Extra Origins 1.19-2
		extrarecords: Extra Records 3.3
			advanced_runtime_resource_pack: Runtime Resource Pack 0.6.2
		fabric: Fabric API 0.58.0+1.19
			fabric-api-base: Fabric API Base 0.4.9+e62f51a3a9
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.7+9ff28f40a9
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.14+b2a4a624a9
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.18+9ff28f40a9
			fabric-command-api-v1: Fabric Command API (v1) 1.2.6+f71b366fa9
			fabric-command-api-v2: Fabric Command API (v2) 2.1.2+0d55f585a9
			fabric-commands-v0: Fabric Commands (v0) 0.2.23+df3654b3a9
			fabric-containers-v0: Fabric Containers (v0) 0.1.28+df3654b3a9
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.2.1+07df213ea9
			fabric-convention-tags-v1: Fabric Convention Tags 1.0.8+37622d24a9
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.3+bd0a0d4aa9
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.1.2+a680b9b4a9
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.28+a6d2f785a9
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.16+9ff28f40a9
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.26+9ff28f40a9
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.26+df3654b3a9
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.19+18990361a9
			fabric-item-api-v1: Fabric Item API (v1) 1.5.5+35a03c43a9
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.26+9ff28f40a9
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.18+9ff28f40a9
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.16+df3654b3a9
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.1.0+33fbc738a9
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.1+03a4e568a9
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.4+9e7660c6a9
			fabric-message-api-v1: Fabric Message API (v1) 1.0.1+513f4a59a9
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.11+33fbc738a9
			fabric-models-v0: Fabric Models (v0) 0.3.15+9ff28f40a9
			fabric-networking-api-v1: Fabric Networking API (v1) 1.1.0+442de8b8a9
			fabric-networking-v0: Fabric Networking (v0) 0.3.15+df3654b3a9
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.0.8+9ff28f40a9
			fabric-particles-v1: Fabric Particles (v1) 1.0.8+dc39553aa9
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.18+23c4cfefa9
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.0.8+9ff28f40a9
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.10+5187d39fa9
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.18+df3654b3a9
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.12+9ff28f40a9
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.5+9ff28f40a9
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.20+df3654b3a9
			fabric-rendering-v1: Fabric Rendering (v1) 1.10.14+9ff28f40a9
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.0.9+e62f51a3a9
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.5.6+5f1a85e0a9
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.24+f05effafa9
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.2.7+9ff28f40a9
			fabric-textures-v0: Fabric Textures (v0) 1.0.18+9ff28f40a9
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.0.9+e62f51a3a9
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.1.1+9e7660c6a9
		fabric-language-kotlin: Fabric Language Kotlin 1.8.4+kotlin.1.7.20
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.7.20
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.7.20
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.7.20
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.7.20
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.18.3
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.4.0
		fabricloader: Fabric Loader 0.14.9
		fallingleaves: Falling Leaves 1.12.2+1.19.2
		figura: Figura 0.1.0-rc.4+1.19
			com_github_moonlight-mc_luaj_luaj-core: luaj-core 3.0.4-figura
			com_github_moonlight-mc_luaj_luaj-jse: luaj-jse 3.0.4-figura
			org_java-websocket_java-websocket: Java-WebSocket 1.5.2
		fwaystones: Fabric Waystones 3.0.2+mc1.19
		iris: Iris 1.2.6
			org_anarres_jcpp: jcpp 1.4.14
		java: OpenJDK 64-Bit Server VM 17
		lambdynlights: LambDynamicLights 2.1.2+1.19
			pride: Pride Lib 1.1.2+1.19
			spruceui: SpruceUI 4.0.0+1.19
		libra: Libra 0.5.4
		mcda: MC Dungeons Armors 2.2.0
		mcdw: MC Dungeons Weapons 5.0.4-1.19
			enchant_giver: Enchant Giver 1.3.0
			reach-entity-attributes: Reach Entity Attributes 2.3.0
		minecraft: Minecraft 1.19
		modmenu: Mod Menu 4.0.4
		moonlight: Moonlight 1.19-2.0.15
		morediscs: More Music Discs 1.19-15
		origins: Origins 1.6.2
			apoli: Apoli 2.5.1
				additionalentityattributes: Additionalentityattributes 1.0.0+1.19
				calio: Calio 1.7.0
				playerabilitylib: Pal 1.6.0
		origins-classes: Origins: Classes 1.4.0
		pehkui: Pehkui 3.4.2+1.14.4-1.19.2
			kanos_config: Kanos Config 0.3.1+1.14.4-1.19
		player_roles: Player Roles 1.6.0
			more_codecs: More Codecs 0.2.2+1.18
			player_roles_api: Player Roles API 1.6.0
		roughlyenoughitems: Roughly Enough Items 9.1.550
			error_notifier: Error Notifier 1.0.9
		sodium: Sodium 0.4.2+build.16
			org_joml_joml: joml 1.10.4
		supplementaries: Supplementaries 1.19-2.0.13
		tooltipfix: ToolTip Fix 1.0.5-1.19
		toomanyorigins: TooManyOrigins 0.6.3+1.19-fabric
			apugli: Apugli 1.8.2+1.19-fabric
				com_github_llamalad7_mixinextras: MixinExtras 0.0.11
		torohealth: ToroHealth Damage Indicators 1.19-fabric-1
		universal-graves: Universal Graves 2.1.0+1.19
			common-protection-api: Common Protection API 1.0.0
			fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
			hologram-api: Hologram API 0.2.2+1.19
			placeholder-api: Placeholder API 2.0.0-beta.6+1.19
			polymer: Polymer (Core) 0.2.2+1.19
				packet_tweaker: Packet Tweaker 0.3.0+1.18.2
				polymer-registry-sync-manipulator: Polymer Registry Sync Manipulator 0.0.1+1.19
			server_translations_api: Server Translations API 1.4.16+1.19
			sgui: sgui 1.1.1+1.19
		voidz: VoidZ 1.0.8
		xaerominimap: Xaero's Minimap 22.12.0
		xaeroworldmap: Xaero's World Map 1.26.1
		xlpackets: XLPackets 2.1-1.17.1
	Loaded Shaderpack: (off)
	Launched Version: fabric-loader-0.14.9-1.19
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce GTX 960/PCIe/SSE2 GL version 3.2.0 NVIDIA 517.48, NVIDIA Corporation
	Window size: 1920x1057
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: Fabric Mods
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz```

**Comment by ChrysanthCow** [comment]
Should be easy enough

**Comment by ChrysanthCow** [comment]
Should be fixed in v1.9.3
